### PR TITLE
102 and 103 ready for use

### PR DESCRIPTION
**Pull request by depa02**

### CODE REVIEW

File: features/steps/101.Getting_Started_steps.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def http_code_is_returned(context, status_code, response):
     stdout.write(f'{diff}\n\n')
 
     assert_that(diff.to_dict(), is_(dict()),
-                f'Response from CB has not got the expected HTTP response body:\n  {diff}')
+                f'(depa) Response from CB test execution does not match the tutorial expected HTTP response body:\n  {diff}')
```
Comment (flopezag) [inline features/steps/101.Getting_Started_steps.py]: I think that it is not needed to put (depa) in the message

File: README.md [hunk 2]
```diff
@@ -83,13 +83,13 @@ Feature: test tutorial 101.Getting Started
...
-process is only about the python dependencies, because the python code do not need 
+The recommended installation method is using a virtualenv. Actually, the installation
+process is only about the python dependencies, because the python code do not need
 installation.
 
 1. Clone this repository.
-2. Create the virtualenv: `virtualenv -ppython3.9 venv`
-3. Activate the python environment: `source ./venv/bin/activate`
+2. Create the virtualenv: `virtualenv -ppython3.9 venv`          (depa\'s laptop --> python3 -m virtualenv .venv)
+3. Activate the python environment: `source ./venv/bin/activate` (depa\'s laptop --> source .venv/bin/activate)
```
Comment (flopezag) [inline README.md]: It sounds a problem in the execution of virtualenv, I suggest to check the permissions of the script. Regarding the documentation, I think that it is not needed to put that comments

    (depa\'s laptop --> python3 -m virtualenv .venv)
    (depa\'s laptop --> python3 -m virtualenv .venv)

**Review by flopezag — CHANGES_REQUESTED** [review]

File: features/steps/101.Getting_Started_steps.py (hunk 1 above)
Comment (depa02) [reply features/steps/101.Getting_Started_steps.py]: agree

File: README.md (hunk 2 above)
Comment (depa02) [reply README.md]: i need them. so perhaps we may say:
"alternatively steps 2 and 3 can be executed as:
2. Create the virtualenv: `python3 -m virtualenv .venv`
3. Activate the python environment: `source .venv/bin/activate`"

what do you think?

Comment (flopezag) [reply README.md]: I think that this a problem in the configuration of virtualenv in your local environment, but it should not be detailed in the README.

File: features/steps/101.Getting_Started_steps.py (hunk 1 above)
Comment (flopezag) [reply features/steps/101.Getting_Started_steps.py]: still pending to make a new commit of the file without this text

**Review by flopezag — CHANGES_REQUESTED** [review]

**Review by flopezag — APPROVED** [review]

**Pull request merged**